.github/workflows/nightly.yml

```diff
@@ -124,29 +160,39 @@ jobs:
           SECRETS_CONTEXT: ${{ toJson(secrets) }}
 
       
-      - name: Publish nightly client to rubygems
-        run: .ci/scripts/publish_client_gem.sh
-        shell: bash
-      
+      - name: Download Ruby client
+        uses: actions/download-artifact@v2
+        with:
+          name: python-client.tar
+
+      - name: Untar Ruby client packages
+        run: tar -xvf ruby-client.tar
 
+      - name: Publish client to rubygems
+        run: bash .github/workflows/scripts/publish_client_gem.sh
       
-      - name: Publish nightly client to pypi
-        run: .ci/scripts/publish_client_pypi.sh
-        shell: bash
+
       
+      - name: Download Python client
+        uses: actions/download-artifact@v2
+        with:
+          name: python-client.tar
+
+      - name: Untar python client packages
+        run: tar -xvf python-client.tar
 
+      - name: Publish client to pypi
+        run: bash .github/workflows/scripts/publish_client_pypi.sh
       
-      - name: Publish nightly docs
-        run: .ci/scripts/publish_docs.sh nightly ${GITHUB_REF##*/}
+
       
+      - name: Download built docs
+        uses: actions/download-artifact@v2
+        with:
+          name: docs.tar
 
-      - name: After failure
-        if: failure()
+      - name: Publish docs to pulpproject.org
         run: |
-          http --timeout 30 --check-status --pretty format --print hb http://pulp/pulp/api/v3/status/ || true
-          docker images || true
-          docker ps -a || true
-          docker logs pulp || true
-          docker exec pulp ls -latr /etc/yum.repos.d/ || true
-          docker exec pulp cat /etc/yum.repos.d/* || true
-          docker exec pulp pip3 list
```

Comment:
Why was the after failure removed? No longer needed?

Reply:
The publish step no longer starts up a container. It only publishes the things that were already generated in previous steps.

Reply:
Ok cool, I'll merge it.